components/org.wso2.carbon.identity.application.authenticator.samlsso/src/test/java/org/wso2/carbon/identity/application/authenticator/samlsso/logout/response/SAMLLogoutResponseFactoryTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.authenticator.samlsso.logout.response;
+
+import org.mockito.Mock;
+import org.powermock.modules.testng.PowerMockTestCase;
+import org.testng.annotations.Test;
+import org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityResponse;
+
+import static org.powermock.api.mockito.PowerMockito.when;
+import static org.testng.Assert.assertTrue;
+import static org.wso2.carbon.identity.application.authenticator.samlsso.TestConstants.IDP_URL;
+import static org.wso2.carbon.identity.application.authenticator.samlsso.TestConstants.INBOUND_SESSION_INDEX;
+import static org.wso2.carbon.identity.application.authenticator.samlsso.TestConstants.HTTP_POST_PARAM_SAML2_RESP;
+
+/**
+ * Unit test cases for SAMLLogoutResponseFactory
```

Comment:
Add full-stop for comments. Check in other places as well

Reply:
Fixed with PR #133